### PR TITLE
Hide CPU info and encode settings with --quiet

### PR DESCRIPTION
**Pull request by shssoichiro**

### CODE REVIEW

File: src/bin/rav1e-ch.rs [hunk 1]
```diff
@@ -508,30 +508,32 @@ fn run() -> Result<(), error::CliError> {
...
-    video_info.bit_depth
-  );
-  info!("Encoding settings: {}", cli.enc);
-
   let tiling =
     cfg.tiling_info().map_err(|e| e.context("Invalid configuration"))?;
-  if tiling.tile_count() == 1 {
-    info!("Using 1 tile");
-  } else {
+  if cli.verbose != Verbose::Quiet {
```
Comment (tdaede) [inline src/bin/rav1e-ch.rs:513]: stylistically I might like cli.verbose > Verbose::Quiet more

**Review by tdaede — APPROVED** [review]

File: src/bin/rav1e-ch.rs (hunk 1 above)
Comment (shssoichiro) [reply src/bin/rav1e-ch.rs:513]: I agree, I might make a followup PR to clean up usages of this throughout the code.

**Review by lu-zero — APPROVED** [review]
The code is fine, but there is a spurious merge from github.

**Comment by shssoichiro** [comment]
I squashed it out, so no worries there.

**Pull request merged**